src/screens/Account/AccountActions.js

```diff
@@ -54,7 +55,7 @@ export default function AccountActions({ account, parentAccount }: Props) {
 
   const onNavigate = useCallback(
     (name: string, options?: NavOptions) => {
-      setDisplayedActions();
+      setNext();
```

Comment:
mmh i'm not sure to understand what this do. next is always undefined, a setNext(undefined) is a noop, react won't do anything because it's === 🤔